addons/zylann.hterrain/native/quad_tree_lod_generic.gd

```diff
@@ -115,7 +109,7 @@ func _update(quad: Quad, lod: int, view_pos: Vector3):
 			# Split
 			quad.children = [null, null, null, null]
 
-			for i in 4:
+			for i in range(4):
```

Comment:
`range` is not necessary

Reply:
I did this only for consistency. The file had 3 methods of iteration: `4`, `range(4)`, and `quad.children.size()`. I could change them all to `4`.